stylesheet.css

```diff
@@ -0,0 +1,257 @@
+body {
+  height: 100vh;
+  width: 100vw;
+  margin: 0;
+  background-image: url("images/galaxy.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center center;
+  font-family: "Space Mono", monospace;
+}
+.header {
+  color: white;
+  text-align: center;
+  font-family: "Space Mono", monospace;
+  font-size: 5rem;
+}
+
+.selectPlayer {
+  color: white;
+  text-align: center;
+  font-family: "Space Mono", monospace;
+  font-size: 2em;
+}
+
+#startButton {
+  /* background-color:white; */
+  background: transparent;
+  color: white;
+  font-size: 1.5rem;
+  padding: 0.6rem 1rem;
+  margin-bottom: 1.5rem;
+  cursor: pointer;
+}
+
+#startButton:hover {
+  color: black;
+  background: white;
+}
+
+.main-container {
+  height: 100%;
+  width: 100%;
+  /* background-image: url("images/galaxy.jpg"); */
+  background-position: center center;
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.game-board {
+  width: 80vmin;
+  height: 80vmin;
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5));
+  cursor: pointer;
+}
+
+.cell-container {
+  color: #6699f6;
+  border: 1px solid currentColor;
```

Comment:
`color` as a CSS property applies affects the value of multiple properties (hence why it is color and not "font-color"). Properties that use `color` such as border, and text decoration, have a _default_ value of `currentColor`, and all `currentColor` does is look at the value of `color`. So you really don't need to specify that here.